DataLayer.Tests/GameTests.cs

```diff
@@ -0,0 +1,45 @@
+﻿using DataLayer.DataLayer.Model;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DataLayer.Tests
+{
+    class GameTests
```

Comment:
Are you really testing if setters and getters working? Your controller should check for stuff like `Name` not being empty or being longer than X when creating.